### PR TITLE
css-brochure-form - bernardino alvarez

### PR DESCRIPTION
**Pull request by bernardino-www**
Screen Shot 2021-08-18 at 10 44 05 PM

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+html,
+body {
+  margin: auto;
+  max-width: 415px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:4]: You shouldn't limit the width of the `html` or `body` elements, they should be allowed to take whatever space is needed. Instead, you would be better off setting a size on the `form` element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! One note for you:

File: css-brochure-form/styles.css (hunk 1 above)
Comment (bernardino-www) [reply css-brochure-form/styles.css]: Good point - just updated

**Review by codingcodymiller — APPROVED** [review]
Great work!! 🎉

**Pull request merged**